README.md

```diff
@@ -53,9 +53,10 @@ This documentation provides a comprehensive overview of the UKEF Digital TradeFi
 6. Set UKEF TFM environment variables in your terminal: `UKEF_TFM_API_SYSTEM_KEY` and `UKEF_TFM_API_REPORTS_KEY`.
 7. Run `npm run env:copy` to copy your root .env file into all the individual projects that need it.
 8. Run `npm ci` in the root folder of the repository. (note: this will install dependencies for the entire project, including those specified in sub-packages. More details on this in the [npm workspaces](./doc/npm-workspaces.md) docs)
-9. Start your local environment with `npm run start`.
+9. Start your local environment with `docker-compose up --build`.
 10. Run migrations on the MSSQL Server database (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for details)
-11. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md))
+11. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md)).
```

Comment:
I am also finding this confusing / hard to read/follow.



- step 10 says "run MSSQL migrations - see X documentation"
- step 11 says "should generate mocks for both Mongo and MSSQL"
- then follows by "to run MSSQL migrations", can be done by running the command ...
- "this creates SQL database tables"

I think something like this (simple example) would be clearer:


- 9. Start your local environment with `docker-compose up --build`
- 10. Load mock MongoDB data with `npm run load`  (does this do _some things_ mock SQL data? Another comment mentions the migration needs to be run to setup some SQL tables)
- 11. At this stage, to run TFM "Utlisation reports" functionality locally, some SQL data migrations need to  be run
  - This is because ....
  - To run migrations, execute `npm run ...` in X directory